pilot/pkg/bootstrap/server.go

```diff
@@ -66,6 +67,7 @@ import (
 	"istio.io/istio/pkg/kube/inject"
 	"istio.io/istio/pkg/security"
 	"istio.io/istio/pkg/spiffe"
+	"istio.io/istio/pkg/test/util/retry"
```

Comment:
we cannot import `istio.io/istio/pkg/test` in production code - I thought we had tests to block this but I guess its not

Reply:
on second thought, the retry isn't required. Although the CA code has a similar logic, it isn't actively being used. Will remove usage of this for now. However, this library is super useful. Strong +1 to move into a common utility package.